### PR TITLE
kernel: clean 4.19 config

### PR DESCRIPTION
**Pull request by abajk**
These symbols exist in kernel version <= 4.15

[CONFIG_FIRMWARE_IN_KERNEL](https://cateee.net/lkddb/web-lkddb/FIRMWARE_IN_KERNEL.html)
[CONFIG_ADDRESS_PCI](https://cateee.net/lkddb/web-lkddb/OF_ADDRESS_PCI.html)
[CONFIG_OF_PCI](https://cateee.net/lkddb/web-lkddb/OF_PCI.html)
[CONFIG_OF_PCI_IRQ](https://cateee.net/lkddb/web-lkddb/OF_PCI_IRQ.html)

Signed-off-by: Aleksander Jan Bajkowski <A.Bajkowski@stud.elka.pw.edu.pl>

### CODE REVIEW

**Comment by adrianschmutzler** [comment]
Maybe rephrase your commit title/message slightly to be more precise on what you do:

```kernel: remove non-existant symbols from 4.19 config

These symbols only exist in kernel version <= 4.15```

**Comment by hauke** [comment]
Thank  you for your pull request, it was applied to master.
